src/main/java/dukeApp/files/TaskList.java

```diff
@@ -0,0 +1,129 @@
+        this.aList = aList;
+    }
+
+    public TaskList() {}
+
+    public int getSize() {
+        return aList.size();
+    }
+
+    /**
+     * Prints task in the list
+     */
+    public void printList() {
+        Task t;
+        System.out.println("Here are the tasks in your list:");
+        for (int i=1; i <= aList.size(); i++) {
+            t = aList.get(i-1);
+            System.out.println(i + ". [" +t.getType()+ "][" +t.getStatusIcon()+ "]" +t.getTask());
+        }
+        System.out.println();
+    }
+
+    /**
+     * Actions to be done for delete & done tasks
+     * @param rank task number in the list
+     * @param dd string indicating delete or done action
+     */
+    public void deleteDone(int rank, String dd) {
+        Action action = new Action(rank, aList);
+
+        if (action.checkNum() == 0) {
+            DukeException error = new DukeException();
+            System.out.println(error.outOfBound());
+            System.out.println("");
+        }
+        else {
+            if (dd.equals("delete")) {
+                deleteAction(rank, aList);
+            } else {
+                doneAction(rank, aList);
+            }
+        }
+    }
+
+<<<<<<< HEAD
+    /**
+     * Call to action for the different task type
+     * @param taskType string indicating task type
+     * @param statement task description
+     */
+=======
+    public void find(String statement) {
+        Find f = new Find(statement, aList);
+        ArrayList<Task> matchTask = f.match();
+        Task t;
+        System.out.println("Here are the matching tasks in your list:");
+        for (int i=1; i <= matchTask.size(); i++) {
+            t = matchTask.get(i-1);
+            System.out.println(i + ". [" +t.getType()+ "][" +t.getStatusIcon()+ "]" +t.getTask());
+        }
+        System.out.println();
+    }
+
+    //call to actions for different task type
+>>>>>>> branch-Level-9
+    public void add(String taskType, String statement) {
+        Task t;
+        statement = statement.substring(statement.indexOf(" "));
+        if (!taskType.equals("todo")) {
+            if (taskType.equals("event")) {
+                t = new Event(statement);
+            } else {
+                t = new Deadline(statement);
+            }
+        }
+        else {
+            t = new Todo(statement);
+        }
+        aList.add(t);
+        printAdded(t, aList); //print msg task added
+    }
+
+    /**
+     * Actions to be executed if delete is called
+     * @param rank task nummber in the list
+     * @param tempList task list
+     */
+    public void deleteAction(int rank, ArrayList<Task> tempList) {
+        Delete delete = new Delete(rank, tempList);
+        String deleteAction = delete.printAction();
+        System.out.println(delete.deleteTask());
+        System.out.println(deleteAction);
+        System.out.println("Now you have " + tempList.size() + " tasks in the list.");
+        System.out.println("");
```

Comment:
Why do you need to print a blank line? If you want you could end of the previous one with "tasks in the list.\n"